geth-poa/entrypoint.sh

```diff
@@ -7,24 +7,37 @@ VERBOSITY=${VERBOSITY:-3}
 GETH_DATA_DIR=${GETH_DATA_DIR:-/data}
 GETH_CHAINDATA_DIR="$GETH_DATA_DIR/geth/chaindata"
 GETH_KEYSTORE_DIR="$GETH_DATA_DIR/keystore"
+MEV_COMMIT_GETH_PASSWORD=${MEV_COMMIT_GETH_PASSWORD:-"pwd"}
 CHAIN_ID=$(cat "$GENESIS_L1_PATH" | jq -r .config.chainId)
 RPC_PORT="${RPC_PORT:-8545}"
 WS_PORT="${WS_PORT:-8546}"
+BLOCK_SIGNER_PRIVATE_KEY=${BLOCK_SIGNER_PRIVATE_KEY:-""}
```

Comment:
Is there a scenario where BLOCK_SIGNER_PRIVATE_KEY should be an empty string? Or should we instead exit script in such a case

Reply:
There is a scenario in which it could be empty (if we are passing the keystore folder instead of the private key). I had to set it to some default value, because otherwise the script didn't work.